addons/website/views/snippets/s_references.xml

```diff
@@ -2,11 +2,12 @@
 <odoo>
 
 <template id="s_references" name="References">
-    <section class="s_references pt48 pb32">
-        <div class="container">
-            <h2 style="text-align: center;">Our References</h2>
-            <p style="text-align: center;">We are in good company.</p>
-            <div class="row">
+    <section class="s_references o_cc o_cc1 pt80 pb80">
+        <div class="container" style="text-align: center;">
+            <h2>Partners and references</h2>
+            <p class="lead">Use this section to boost your company's credibility.</p>
+            <a href="#">See our case studies <i class="fa fa-long-arrow-right ms-2"/></a>
+            <div class="row align-items-center justify-content-center mt-4">
                 <div class="col-lg-2 pt16 pb16">
```

Comment:
The images are large when you switch to tablet mode, even in mobile mode in fact.
Perhaps we could use 2 columns for these, what do you think?

```suggestion
                <div class="col-6 col-lg-2 pt16 pb16">
```